packages/core/admin/admin/src/pages/AuthPage/components/Login/BaseLogin.js

```diff
@@ -30,7 +30,7 @@ const Login = ({ onSubmit, schema, children }) => {
           initialValues={{
             email: '',
             password: '',
-            rememberMe: false,
+            rememberMe: true,
```

Comment:
I don't think it is GDPR compliant to do so?

Reply:
Doing this would also have security implications.

Reply:
@pierreburgy lets say you are on a public PC and you just click trough normaly if you closed the tab you would be auto logged out now the next person can go to your strapi instance and change stuff. so for development mode instances yes this would make sence for production it should be a config flag and it should be fully disabled for better security.

Reply:
@Boegie19 isn’t it how it works with other software? I’ve never seen a tool storing the token in sessionStorage only, which results in the need to log in to every browser tab

Reply:
> @Boegie19 isn’t it how it works with other software? I’ve never seen a tool storing the token in sessionStorage only, which results in the need to log in to every browser tab

That is correct but they then limit the cookie length to lets say 1 week. Also that is for social networks.

strapi is meant for companies so hence should be more secure. since you could have sensitive data in your strapi instance.

So I think a feture that allows you to do this is not wrong. but it should not be enabled by default.

Reply:
Doesn’t the JWT token include an expiration date?

Reply:
JWT tokens of strapi are valid for ever. unless you change your password.

Reply:
Technically they are valid for 30 days not forever but I would agree here.

Maybe we need to change the default admin expiration to 7 days 🤔

Reply:
@alexandrebodin what are your thoughts? Happy to change the token duration within this PR, but maybe we should keep the same duration or change it through another PR

Reply:
I can only comment security wise, if we pre check the remember me then we should definitely go for 7days expiration by default and let people do longer or shorter in the config like it's already possible

Reply:
My objection would be you're adding risk for minor convenience ie you log in on a different computer and you're an admin, the next person to use that computer is logged in as you – it's the common reason no one does this by default. Opting in is an explicit action, forgetting to opt in means you have to log in again which is safer.

IMO I wouldn't do this 🙂

Reply:
@joshuaellis on any service, the next person using the computer would get access to the account (except if the previous person logged out). The only difference is that the session would be much shorter.

@alexandrebodin To me, the real issue is that we use, by default, sessionStorage. This means when the user opens a new tab, they have to log in again. The user can even be logged in as two different users in two different tabs. This, in my opinion, is very unusual (I've never seen a service doing this) and gives a poor user experience.

So, the best solution is probably to:
 - Checkbox not checked: localStorage, 24-hour token duration
 - Checkbox check: localStorage, 7-day token duration
 
It would be even better to use sessions through cookies (for security reasons), but that would be a bigger update.

Reply:
The sessionStorage behavior is exaclty what a site with a remember me unchecked would do this was very common in the past. Nowdays nobody does it anymore and just use persistent sessions by default. 

A lot of website do this if we want to go in details on the workflow: 

- By default use a x hours tokens (you can also auto refresh the token if the person is logged in close to the expiration up to 7 days when you force a relogin regardless of what the user did)
- Add a checkbox to allow "staying connected for x days" (e.g payfit: 30days) 

Side note: 
Cookies won't change anything in terms of user experience here.

- A cookie without expiration is expired when the browser is closed the same way a sessionStorage would be clearer.
- A cookie with an expiration will expire on that time like with localStorage

Reply:
A few points that we can be revisited

- the admin was originaly meant to be hosted wherever you want and potentially not on the same domain as the server (like on a CDN) which breaks same domain cookie policy. 
- The api calls where meant to be stateless to avoid having to manage sessions on the server
- To support horizontally scaled apps you need to persist the sessions in DB